pyroute2/iwutil.py

```diff
@@ -51,18 +51,18 @@ def list_wiphy(self):
 
     def get_interface_by_phy(self, attr):
         '''
-        Describe me
+        Get interface by phy name ( use x.get_attr('NL80211_ATTR_WIPHY') )
         '''
         msg = nl80211cmd()
         msg['cmd'] = NL80211_NAMES['NL80211_CMD_GET_INTERFACE']
         msg['attrs'] = [['NL80211_ATTR_WIPHY', attr]]
         return self.nlm_request(msg,
                                 msg_type=self.prid,
-                                msg_flags=NLM_F_REQUEST)
+                                msg_flags=NLM_F_REQUEST| NLM_F_DUMP)
```

Comment:
https://travis-ci.org/svinota/pyroute2/jobs/51269860

Pls don't forget flake8. I use https://github.com/nvie/vim-flake8 for that.